packages/discovery-react-components/src/components/CIDocument/__stories__/default.md

```diff
@@ -0,0 +1,3 @@
+#### Overview
+
+The `CIDocument` component provides Element Classification to the user. It takes a PDF and enriches it with features a customer might deem important (such as Obligation, Deliverables, Assignments, etc...). This would help customers get a better understanding of their document, or help them find what they need in a large document in a more efficient way.
```

Comment:
Bah, didn't get to this in time. Some issues:
- This component doesn't "take" anything. Rather it displays a document enriched by one of the Content Intelligence enrichments.
- Don't use "Element Classification". I don't think we use that anywhere. Also, the description above mostly only applies to contracts. Make it more generic